pkg/controllers/kube-apiserver.go

```diff
@@ -165,7 +165,7 @@ func (s *KubeAPIServer) configure(cfg *config.MicroshiftConfig) error {
 									APIVersion: "route.openshift.io/v1",
 									Kind:       "HostAssignmentAdmissionConfig",
 								},
-								Domain: cfg.Cluster.Domain,
+								Domain: cfg.BaseDomain,
```

Comment:
@benluddy would this be the right thing to use here ^ ?  before we were using cluster.local but I wonder if this is more related to  https://github.com/openshift/microshift/pull/1176/files#diff-c70ba87d9ac52efa40c2eb55913d76e6c263c9690a9f3e48b074865cbd366738L65  ?

Reply:
Missing `cfg.ClusterName`, I assume.

Reply:
I believe it probably makes sense to add a constant "ClusterName" to the config module that we can use instead of just adding "microshift." everywhere. Then the intent will be clearer and also, it will be simpler to convert into cfg.ClusterName afterwards.

Reply:
Or even apps. + ClusterName + BaseDomain? 

I am not sure of the intent of this admission controller config. @benluddy ^